app/services/cache/mapping/MetaDataBuilder.scala

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package services.cache.mapping
+
+import forms.Choice
+import javax.xml.bind.JAXBElement
+import javax.xml.namespace.QName
+import models.declaration.SupplementaryDeclarationData.SchemaMandatoryValues
+import services.cache.ExportsCacheModel
+import services.mapping.declaration.DeclarationBuilder
+import uk.gov.hmrc.http.cache.client.CacheMap
+import wco.datamodel.wco.dec_dms._2.Declaration
+import wco.datamodel.wco.documentmetadata_dms._2.MetaData
+import wco.datamodel.wco.metadata_ds_dms._2.{
+  MetaDataAgencyAssignedCustomizationCodeType,
+  MetaDataResponsibleAgencyNameTextType,
+  MetaDataResponsibleCountryCodeType,
+  MetaDataWCODataModelVersionCodeType,
+  MetaDataWCOTypeNameTextType
+}
+
+object SubmissionMetaDataBuilder {
+
+  def build(cacheModel: ExportsCacheModel): MetaData = {
+    createMetaDataWithConstants
+
+//    val element: JAXBElement[Declaration] = new JAXBElement[Declaration](
+//      new QName("urn:wco:datamodel:WCO:DEC-DMS:2", "Declaration"),
+//      classOf[Declaration],
+//      DeclarationBuilder.build(cacheMap, choice)
+//    )
+//    createMetaDataWithConstants.setAny(element)
+    createMetaDataWithConstants
```

Comment:
You're invoking this method two times. In lines 39 and 47

Reply:
I'll fix this.